README.md

```diff
@@ -107,6 +106,153 @@ print(myShoppingList)
+// They're back, let's make a list!
+
+shoppingList = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you create an array variable called 'shoppingList' that will accept values of type `String`. You then assign five string values to the variable later in the program.
+
+## Constant Arrays
+
+As you have used the `var` keyword in all the examples so far, these collections of values are _mutable_. This means you can add, remove or change items in the array. As with other variable types in Swift, to make an array constant, unchangeable or _immutable_, use the `let` keyword.
+
+For example, a shopping list may change, but the ingredients for a cheese sandwich never will:
+
+```swift
+let cheeseSandwich: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+## Other Values in an Array
+
+As you would expect, arrays can contain values apart from strings, and created the same way. For example, to create an array recording attendance at your Saturday afternoon sandwich meet:
+
+```swift
+var afternoonAttendance: [Int] = [10, 5, 15, 2, 12]
+```
+
+Whilst you can create arrays that contain different values, each array may only contain values of the **same** type. For example, you cannot have an array of mixed number and string values.
+
+Try this array in a Playground and see what happens:
+
+```swift
+var mixedArray = [10, "cheese", 15, 2, "Bread"]
+```
+
+## Accessing Items in an Array
+
+Like any series of items in a list, items in an array have a number associated with them that represents their place within it. In programming, this is called an 'index'. Remember, that whilst we start counting from 1, many programming languages start from 0, making the first item in an array actually 'at index 0'.
+
+Continuing with our example, you want to know what the second item on the shopping list is, with a normal list this would be:
+
+1. Bread
+2. **Butter**
+3. Cheese
+4. Lettuce
+5. Tomatoes
+
+But, as an array starts from 0, to be precise, it's actually:
+
+0. Bread
+1. **Butter**
+2. Cheese
+3. Lettuce
+4. Tomatoes
+
+In the `shoppingList` array, if you want to print the value of the item in the second position, starting from 0, this is position 1\. Assign the value in that position to a variable, and print the value of the variable:
+
+```swift
+let secondItem = shoppingList[1]
+
+print(secondItem)
+// prints "Butter"
+```
+
+The syntax above is called _subscript_. The name of the array is `shoppingList` and you want to access the second item in the array by using square brackets and the position of the value, i.e. `[1]`.
+
+## Changing Items in an Array
+
+You have decided instead to make a ham sandwich, so you need to change the 'Cheese' value in the array. Again you need to know the position of 'Cheese' and change it by assigning a new value.
+
+> What index is the 'Cheese' value at?
+
+Remember, that Swift counts from '0', so the value of 'Cheese' is actually at position '2', three values into the array.
+
```

Comment:
I like how you engage with the student here in a very friendly conversational way.